pattern_library/__init__.py

```diff
@@ -1,3 +1,5 @@
+from .context_modifiers import register_context_modifier  # NOQA
```

Comment:
Can we add an `__all__` rather than using a noqa comment?

Reply:
I'm not so fond of the repetitive nature of `__all__` (it's another thing to maintain), but that's what you prefer, that's all good with me :)

Reply:
Implemented in 8eafff0